target/linux/ramips/dts/mt7621_mikrotik_routerboard-rb760igs.dts

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621_mikrotik.dtsi"
+
+/ {
+	compatible = "mikrotik,rb760igs", "mediatek,mt7621-soc";
```

Comment:
Keeping names consistent is really a pain here, so let's just follow the scheme from ath79 now (which is consistent with model/DEVICE_MODEL in ramips):
DTS file name: mt7621_mikrotik_routerboard-760igs.dts
compatible: mikrotik,routerboard-760igs
model: keep as below.
DEVICE_MODEL: RouterBOARD 760iGS